scripts/scan-g-cloud-services-for-bad-words.py

```diff
@@ -115,17 +119,31 @@ def check_services_with_bad_words(output_dir, framework_slug, client, suppliers,
                         for word in bad_words:
                             if get_bad_words_in_value(word, service.get(key)):
                                 output_bad_words(
-                                    supplier["supplierId"], framework_slug, service["id"],
-                                    service["serviceName"], service["serviceSummary"], key,
-                                    service.get(key), word, writer)
+                                    supplier["supplierId"],
+                                    framework_slug,
+                                    service["id"],
+                                    service["serviceName"],
+                                    service.get("serviceSummary", service["serviceDescription"]),
```

Comment:
If it has neither then maybe it should just die though?  we shouldn't have any services without either key.